### PR TITLE
Update BLorient813.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient813.xml [hunk 1]
```diff
@@ -30,20 +31,272 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+                            <msItem xml:id="ms_i1.49.1">
+                              <title type="complete" xml:lang="en">The list of the patriarchs
+                              of Alexandria</title>
+                              <textLang mainLang="gez"></textLang>
+                              <note>The list ends with <persName ref="PRS6752Mark"></persName>.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.49.2">
+                              <locus from="173v"></locus>
+                              <title type="complete" xml:lang="en">The Seven Wonders of the World</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient813.xml:257]: I see that this is also attested in another BL ʾAbušākər manuscript. Could you find some information on this textual unit? Is it normally attested in ʾAbušākər manuscripts and could be included as a subunit in the work record?

File: LondonBritishLibrary/orient/BLorient813.xml [hunk 2]
```diff
@@ -128,21 +421,27 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                    <term key="Chronography"></term>
                     <term key="ZaMa" cert="low"/>
                     <term key="MoPe" cert="low"/>
                 </keywords>
             </textClass>
-            <langUsage><language ident="en">English</language></langUsage>
+            <langUsage>
+              <language ident="en">English</language>
+              <language ident="gez">Gǝʿǝz</language>
+              <language ident="en">English</language>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient813.xml:433]: Amharic

**Review by thea-m — CHANGES_REQUESTED** [review]
just a question

File: LondonBritishLibrary/orient/BLorient813.xml (hunk 1 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient813.xml:257]: Yes, you are right, Wonders and list of Patriarchs seems to belong to the normal structure. I wull add them to the work record.

**Review by thea-m — APPROVED** [review]

**Pull request merged**